### ISSUE TITLE
Win10: WebUI does not show any peers

### ISSUE THREAD

**Issue by DungeonMasterQ** [ipfs/go-ipfs#4862]
#### Version information: 
go-ipfs version: 0.4.13-
Repo version: 6
System version: amd64/windows
Golang version: go1.9.2

#### Type:
Bug

#### Description:
* What you were doing when you experienced the bug?
After running **ipfs init** then **ipfs swarm peers** I saw a numerous amount of peers listed
After running **ipfs daemon** then navigating to the WebUI, the Connections tab showed 0 peers connected
* Any error messages you saw, *where* you saw them, and what you believe may have caused them (if you have any ideas).
I have experienced error messages in the CLI window that is being used to run the daemon as such:
"[0;37m14:30:29.579 [31mERROR [0;34m   bitswap: [0mError writing block to datastore: Access is denied. [0;37mbitswap.go:317[0m"
I believe these errors may have something to do with the issue because when I re run the daemon and DO NOT close the webui, some of the peers do show up - however, if I navigate to a different tab, then back to the connections tab, there are 0 peers again and I experience more errors. I cannot resolve the issue unless I stay on the connections tab and re run the daemon.

This does not affect much functionality other than usability - I'm still able to access files I have added with ipfs over an internet connection - this is more of a UI bug IMHO

**Comment by leerspace** [comment]
The errors in your second bullet point are likely a duplicate of ipfs/go-ipfs/issues/3971.

**Comment by leerspace** [comment]
For the WebUI not showing any connections, how long are you waiting on the page to see if it changes? It's a pretty intensive page that takes a while to show something other than 0 peers on my slow laptop.
